README.md

```diff
@@ -128,6 +129,7 @@ See code for all available configurations.
 | [Tuxedo InfinityBook v4][]        | `<nixos-hardware/tuxedo/infinitybook/v4>`          |
 
 [Acer Aspire 4810T]: acer/aspire/4810t
+[Asus TUF FX504GD]: asus/fx504gd
```

Comment:
It's not yet documented but can you also add it here? https://github.com/NixOS/nixos-hardware/blob/master/flake.nix

Reply:
Is it good ? I kind of think I messed up a little bit by trying to commit and squash, tell me if I need to change something.

Reply:
I can squash on merge.